README.md

```diff
@@ -0,0 +1,40 @@
+# GDX Analytics Looker Google MyBusiness
+
+GDX Analytics Looker project to integrate the Google MyBusiness API for Location Insights
+
+## Features
```

Comment:
Can you rephrase this to describe the the features of this project?

Reply:
Done.